translate: tools/index.md + tools/editor/* (#128)
OI-wiki/i18n: docs/tools/index.md

> This chapter will introduce tools and softwares related to competitive programming, including some introductions of code editor and tools about OJ.

Reviewer: ```suggestion
This chapter will introduce tools and softwares related to competitive programming, including some introductions of code editor and tools for OJ's.
```

> Program is a tool solving problems in competitive programming, and proficiency with code editors is a prerequisite for learning knowledge of competitive programming.

Reviewer: ```suggestion
Programming is a tool to solve problems in competitive programming, and proficiency with code editors is a prerequisite for learning knowledge of competitive programming.
```

> Understanding tools relating to OJ will make your journey of competitive programming easier.

Reviewer: ```suggestion
Knowing tools for OJ's will make your journey of competitive programming easier.
```
